0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -587,6 +599,10 @@ The `Message` class is used to describe local ledger message. All fields are req
 | `String` | [ilp](#messageilp) | Base64-encoded ILP packet |
 | `Object` | [custom](#messagecustom) | Object containing ledger plugin specific options |
 
+Either `ilp` or `custom` (but not both) should always be included.
```

Comment:
But it's not something we're currently using, right? So you want to keep that for future extensibility, when people want to define request types that are partially-OER and partially-JSON?